test/opentracing_compatibility.js

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import apiCompatibilityChecks from 'opentracing/lib/test/api_compatibility.js';
+import * as apiCompatibilityChecks from 'opentracing/lib/test/api_compatibility.js';
```

Comment:
`apiCompatibilityChecks` is a default so that makes this alias incorrect unless you use `apiCompatibiltiyChecks.default()` below: https://github.com/opentracing/opentracing-javascript/blob/master/src/test/api_compatibility.ts#L92